ui/src/app/shared/components/webcomponents/upload-entities-modal.component.ts

```
@@ -10,17 +12,43 @@ import {TestDevice} from "../../../models/test-device.model";
     let entityUrl;
     if(this.modalData.linkedEntityList['cachedItems'][0] instanceof TestDevice)
       entityUrl = "/ui/td/plans/"+ id +"/details";
     window.open(window.location.origin + entityUrl, "_blank");
   }
+
+  private setTestPlanNames(content: TestPlan[]) {
+    function findAssociatedTestPlan(d) {
+      return  content.find((t)=> d.testPlanId==t.id)
+    }
+    this.modalData.linkedEntityList.cachedItems.forEach(
+      (d,i)=> this.modalData.linkedEntityList.cachedItems[i].name = findAssociatedTestPlan(d).name)
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)